addons/account/tests/account_test_classes.py

```diff
@@ -21,15 +22,73 @@ def setUp(self):
             _logger.warn('Test skipped because there is no chart of account defined ...')
             self.skipTest("No Chart of account found")
 
-    def check_complete_move(self, move, theorical_lines):
+    def check_complete_move(self, move, theorical_lines, fields_name=None):
```

Comment:
sorry, I was not clear on my first comment :-s

theorical_lines should be a list of **dict**, and move should be a recordset of account.move.line, that would be much more convenient.

The idea is, in a next iteration, to remove codes doing similar thing such as `check_results()` in t`est_reconciliation.py` or `check_journal_items()` in `test_payment.py`

Actually, the parameter `fields_name` is useless as it can be deducted from `theorical_lines[0].keys()`

Can you change that?

Reply:
stuff moved to https://github.com/odoo/odoo/pull/25129